test-helpers/src/docker_compose.rs

```diff
@@ -138,8 +163,7 @@ impl DockerCompose {
     /// # Panics
     /// * If `count` occurrences of `log_text` is not found in the log within 90 seconds.
     ///
-    fn wait_for_log(&self, log_text: &str, count: usize) {
-        let timeout_seconds = 110;
```

Comment:
Can we just bump this to the safer 150 instead of allowing the choice?
If you really want to allow the choice per docker-compose.yaml then we should specify the value within the wait_for_containers_to_startup match, thats kind of its point after all.

Reply:
It will save us 40 seconds in some cases so I think having it as low as possible is good. 

I can move the logic to `wait_for_containers_to_startup`.